server/common/common/settings.py

```diff
@@ -0,0 +1,147 @@
+"""
+Django settings for common project.
+
+Generated by 'django-admin startproject' using Django 2.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+import pymysql
+
+pymysql.install_as_MySQLdb()
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'lb%h0=0@4txgkqbp4nwgfpk$yvwh6l6@%y!#kwe9l2)02wb#8l'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
These values should not be checked in, but should be set only on your local client, for debugging purposes.